### ISSUE TITLE
zfcuser_login_form not found

### ISSUE THREAD

**Issue by pensiero** [ZF-Commons/ZfcUser#612]
After updating ZfcUser, I received this error:
```
Zend\ServiceManager\ServiceManager::get was unable to fetch or create an instance for zfcuser_login_form
```

Is the name changed?

**Comment by pensiero** [comment]
Uh, I believe that now is `->get('FormElementManager')->get('zfcuser_login_form')`, is it correct?

**Comment by nepda** [comment]
See #610 , yes. In version 1.4.1 this issue should be fixed. If this is your own code, raising this error, you were right. Just use `->get('FormElementManager')->get('zfcuser_login_form')`.

**Comment by pensiero** [comment]
Thanks @nepda . 1.4.1 is backward compatible with `->get('FormElementManager')->get('zfcuser_login_form')`?

**Comment by Danielss89** [comment]
This is a BC break. I will release a version later today which should fix it.

**Comment by pensiero** [comment]
:+1: thanks

**Comment by Goldfinch** [comment]
do not forget to fix also in service
https://github.com/ZF-Commons/ZfcUser/blob/1.x/src/ZfcUser/Service/User.php#L207
otherwise, we get an error when we try to registration

**Comment by N3XT0R** [comment]
Hello Guy, when it is a breaking change, so please release it conformally to the semantic versioning standard.

**Comment by Danielss89** [comment]
@N3XT0R We follow semantic versioning, but sometimes a BC break finds it way anyway. That's why i'll release a new version fixing this.

**Comment by Danielss89** [comment]
Thanks. Should be fixed in newly released version 1.4.2.
https://github.com/ZF-Commons/ZfcUser/releases/tag/1.4.2
